src/Airship/Types.hs

```diff
@@ -101,19 +104,21 @@ data ResponseBody
 escapedResponse :: Text -> ResponseBody
 escapedResponse = ResponseBuilder . fromHtmlEscapedText
 
-data Response = Response { _responseStatus     :: Status
-                         , _responseHeaders    :: ResponseHeaders
-                         , _responseBody       :: ResponseBody
+data Response = Response { _responseStatus  :: Status
+                         , _responseHeaders :: ResponseHeaders
+                         , _responseBody    :: ResponseBody
                          }
 
-data ResponseState = ResponseState { stateHeaders   :: ResponseHeaders
-                                   , stateBody      :: ResponseBody
-                                   , _params        :: HashMap Text Text
-                                   , _dispatchPath  :: [Text]
+data ResponseState = ResponseState { stateHeaders  :: ResponseHeaders
+                                   , stateBody     :: ResponseBody
+                                   , _params       :: HashMap Text Text
+                                   , _dispatchPath :: [Text]
                                    }
 
 type Trace = [Text]
 
+type ErrorResponses = Map HTTP.Status (ByteString, ResponseBody)
```

Comment:
Also can we use a `HashMap` here instead of a Map? Would kill our extra `containers` dependency, though not by much.